Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
+    {
+        if (defined('PHP_QUERY_RFC3986')) {
+            // add htmlspecialchars because twig add it auto
```

Comment:
If you want show some html code you need using `raw` option otherwise it will replace:
```
{{ {foo: "bar", number: 3}|url_encode }}
{{ {foo: "bar", number: 3}|url_encode|raw }}
The results:
foo=bar&amp;number=3
foo=bar&number=3

```